### PR TITLE
Removing outdated information on automatic merge process

### PR DESCRIPTION
**Pull request by ThomasLandauer**
See https://github.com/symfony/recipes-contrib/issues/1076#issuecomment-758502196

| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,25 +19,7 @@ For more information about contributing a recipe, read the `documentation`_ on
...
-
-* The pull request author or the reviewer is a "trusted user". A "trusted user"
-  being one of the following:
-
-  * A Symfony Core Merger;
-
-  * The user is the Github owner for all modified packages;
-
-  * The user is a member of the organizations (publicly visible) for all
-    modified packages.
```
Comment (fabpot) [inline README.rst:40]: Actually, everything is still true except for the last item which should read: "The pull request author or the reviewer is a Symfony Core Merger"

**Review by fabpot — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (ThomasLandauer) [reply README.rst:40]: This is already the first point ;-) So in fact, the third needs to be deleted - that's what I did now.

Comment (fabpot) [reply README.rst:40]: Why keep a list with one item?

Comment (ThomasLandauer) [reply README.rst:40]: *Two* items (GitHub's display here is a little confusing):
  * a Symfony Core Merger;
  * the Github owner for all modified packages.

Comment (fabpot) [reply README.rst:40]: The second one is not true anymore; that's why I proposed "The pull request author or the reviewer is a Symfony Core Merger"

Comment (ThomasLandauer) [reply README.rst:40]: Sorry, didn't understand what you meant by "last item" ;-) Fixed now - hopefully :-)

**Review by fabpot — APPROVED** [review]

**Pull request merged**